### DISCUSSION TITLE
Restart DMS when server reboots: best practice?

### DISCUSSION THREAD

**Discussion by g40** [docker-mailserver/docker-mailserver#4195] [Q&A] [answered]
SO has this old answer: https://stackoverflow.com/questions/30449313/how-do-i-make-a-docker-container-start-automatically-on-system-boot

Is there a better way these days? DMS is running on a Linode instance using Ubuntu24.04 and DMS 14.0.0. 

Any sane suggestions welcomed. :)

**Comment by bocan** [comment, marked as answer]
As it's 2024, why not use `docker compose` rather than `docker`?  Then you'd just ensure you add this line to the compose file.

```
    restart: always
```

I run many other things in docker compose on a AWS SPOT instance.  It can vanish and return as much as it wants and there's no need to restart anything.
